app/src/main/res/values/strings.xml

```diff
@@ -416,5 +416,6 @@
     Enter a ratio in the form x:y.
   </string>
 
-
+  <string name="coming_soon">Coming Soon</string>
```

Comment:
Can we have a Home section as we have sections like `<!-- ProfileChooserFragment -->`

Reply:
This will be updated in UI PR. I will revert this change.